models/maespa/R/met2model.MAESPA.R

```diff
@@ -138,14 +138,20 @@ met2model.MAESPA <- function(in.path, in.prefix, outfolder, start_date,
        defaultCO2 = 400 #400 is estimation of atmospheric CO2 in ppm)
      } else {
        CA <- CA*1e6
+       defaultCO2 = 400 #Set variable but it will not be used as CA is available
```

Comment:
There's no reason to define the same constant twice in separate parts of this if statement